include/superior_mysqlpp/prepared_statement.hpp

```diff
@@ -185,42 +274,72 @@ namespace SuperiorMySqlpp
             }
         }
 
+        /**
+         * Accessor of tuple containing storage of param bindings.
+         * Mutable variant.
+         */
         auto& getParams() noexcept
         {
             return paramsBindings.data;
         }
 
+        /**
+         * Accessor of tuple containing storage of param bindings.
+         * Const variant.
+         */
         const auto& getParams() const noexcept
         {
             return paramsBindings.data;
         }
 
+        /**
+         * Accessor of tuple containing storage of result bindings.
+         * Mutable variant.
+         */
         auto& getResult() noexcept
         {
             return resultBindings.data;
         }
 
+        /**
+         * Accessor of tuple containing storage of result bindings.
+         * Const variant.
+         */
         const auto& getResult() const noexcept
         {
             return resultBindings.data;
         }
 
+        /**
+         * Convenience function to replace current query params.
+         * @param args Forwarded new data,
+         */
         template<typename... Args>
         void setParams(Args&&... args)
         {
             paramsBindings.data = std::forward_as_tuple(std::forward<Args>(args)...);
         }
 
+        /**
+         * Returns size of parameterBindings.
```

Comment:
It is count indeed. IMO, because in C++ it is common that size() returns the amount of elements for std containers, I consider size and count to be equivalent terms. Are you considering ambiguity with `sizeof`-like semantics?